network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/PhaseTapChangerAdderImpl.java

```diff
@@ -26,6 +26,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(PhaseTapChangerAdderImpl.class);
 
+    private final TapChangerParent tapChangerParent;
```

Comment:
tapChangerParent seems not being used

Reply:
It is used in the add method, when creating the PhaseTapChangerImpl.